Firestore/core/src/firebase/firestore/model/field_value_options.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_MODEL_FIELD_VALUE_OPTIONS_H_
+#define FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_MODEL_FIELD_VALUE_OPTIONS_H_
+
+namespace firebase {
+namespace firestore {
+namespace model {
+
+/** Defines the return value for pending server timestamps. */
+enum class ServerTimestampBehavior { None, Estimate, Previous };
```

Comment:
I think these are supposed to be styled like constants, i.e. `kNone`. (http://go/cstyle#Enumerator_Names). I don't recall if the rest of our codebase follows or ignores this though.

Reply:
I think we do sometimes and sometimes not.

I keep forgetting to do it because I think it's so ugly. Thanks for the reminder :-).